build.sbt

```diff
@@ -20,25 +20,24 @@ startYear in ThisBuild := Some(2018)
 description in ThisBuild := "A library for working with standards from the EPC extension to the GS1 standard"
 crossPaths in ThisBuild := false
 
-bintrayOrganization in ThisBuild := Some("nike")
-bintrayRepository in ThisBuild := "maven"
-bintrayPackage in ThisBuild := "epc-standards"
-
-version in ThisBuild := sys.env.get("TRAVIS_TAG").filter(_.trim.nonEmpty).getOrElse("0-SNAPSHOT")
+publishTo in ThisBuild := {
+  val repo = "https://artifactory.nike.com/artifactory/maven"
+  if (isSnapshot.value) {
```

Comment:
Fun fact, sbt determines `isSnapshot` by checking if the version ends in "-SNAPSHOT"